src/rgw/rgw_process.cc

```diff
@@ -461,7 +461,7 @@ int process_request(rgw::sal::Store* const store,
   if (latency) {
     *latency = lat;
   }
-  dout(1) << "====== req done req=" << hex << req << dec
+  dout(2) << "====== req done req=" << hex << req << dec
```

Comment:
i know this line has been useful in the past, because it includes both the `http_status` and the `op status`. the beast frontend's log line only includes the http status

so at the default log level, we'll see the http error like `403` but lose the ability to distinguish between `AccessDenied`, `SignatureDoesNotMatch`, `QuotaExceeded`, etc

Reply:
I'm not sure that argues against moving it to loglevel 2

Reply:
you want to change the error prints, tho?